src/schema/ghga.yaml

```diff
@@ -1626,3 +1691,21 @@ enums:
       vcf:
         description: >-
           vcf File for storing gene sequence variations.
+
+  submission_status_enum:
+    description: >-
+      Enum to capture submission status.
+    permissible_values:
+      in progress:
+        description: Signifies that a submission is in progress.
+      submitted:
+        description: Signifies that a submission is complete and submitted.
+
+  release_status_enum:
```

Comment:
Maybe in the future we could add an "embargo" property, that indicates when a researcher wants to hold back data for a certain time

Reply:
I think embargo is another way of looking at `submission status` and `release status` that is communicated from the data submitter to GHGA. A separate property would be rarely used and might just be duplicative.